backend/drive/drive.go

```diff
@@ -829,11 +831,27 @@ func (f *Fs) list(ctx context.Context, dirIDs []string, title string, directorie
 	if filesOnly {
 		query = append(query, fmt.Sprintf("mimeType!='%s'", driveFolderType))
 	}
+
+	// Constrain query using filter if this remote is a sync/copy/walk source.
+	if fi, use := filter.GetConfig(ctx), filter.GetUseFilter(ctx); fi != nil && use {
+		if !fi.ModTimeFrom.IsZero() {
+			query = append(query, fmt.Sprintf("(modifiedTime >= '%s' or mimeType = '%s')",
+				fi.ModTimeFrom.Format("2006-01-02T15:04:05"), driveFolderType))
```

Comment:
Is there no timezone in this time? Or is it guaranteed to be UTC? EIther way this doesn't seem quite right.

Reply:
Ouch! My bad!

Reply:
PTAL